### DISCUSSION TITLE
When does the sorc do diablo/baal

### DISCUSSION THREAD

**Discussion by Wullff91** [isid0re/SoloLeveling#215] [Q&A] [answered]
Hey my sorc is now 86 and just doing act1-act 3 and reapting it but not finished act 4 & act 5 until now
He do Templeruns until forgotten Temple use tp and game crashes.
 Window has unexpectedly exited... starting profile

When should she do it?
where can i see which gear is needed or where can i see her route ?

**Comment by Wullff91** [comment]
Ok my second sorc does the same start it to check it.
starts with pits, summoner, ancitens and then act3 temple runs.
Ends in forgotten temple go to town and game chrashes.

My hammerdin do the same after he fineshed until act 5
Just pits until temple runs and quits after forgotten temple.

**Comment by isid0re** [comment, marked as answer]
Are you using the most recent repo? There was a fix for a previous problem with act 3 sewer recently.

**Reply by Wullff91** [reply]
Yeah got the news version.

sewers went fine now.

But the sorc just do Pits, summoner, ancients tunnels and temple runs and game crashes after the The forgotten temple.

Just try a druid and see whats happens.

**Reply by isid0re** [reply]
Temple run crashes have been fixed

**Reply by Wullff91** [reply]
yeah they do act 4 now looks like its working and no crashes so far
